wagtail/contrib/forms/tests/test_views.py

```diff
@@ -361,6 +362,52 @@ def setUp(self):
         # Login
         self.login()
 
+    def get_url(self, viewname, *args, **kwargs):
+        """
+        Reverse a URL and append querystring arguments.
+        """
+        querystring = kwargs.pop("querystring", None)
+        url = reverse(viewname, args=args, kwargs=kwargs)
+        if querystring:
+            url += "?" + urlencode(querystring)
+        return url
+
+    def test_export_urls_include_filters(self):
```

Comment:
This test can be simplified by doing the `reverse()` once and adding the format parameters within a `subTest`, e.g.

```py
from django.utils.html import escape
from django.utils.http import urlencode

# ...

    def test_export_urls_include_filters(self):
        list_url = reverse("wagtailforms:list_submissions", args=(self.form_page.id,))

        # Ensure that the download URLs include the filter parameters
        response = self.client.get(list_url, {"name": "Alice"})
        for format in ("xlsx", "csv"):
            with self.subTest(format=format):
                params = urlencode({"name": "Alice", "export": format})
                expected_url = f"{list_url}?{params}"
                self.assertContains(response, escape(expected_url))
```

Reply:
Also, it would be great if we had another test to verify that the downloaded file only contains the filtered items (see tests that use `load_workbook` from `openpyxl`). It's slightly more complicated though, and it's technically already covered in some other tests, so I don't mind if we don't have it.